### ISSUE TITLE
Error loading admin page after upgrade to 1.6.8.

### ISSUE THREAD

**Issue by withindale** [getgrav/grav#2475]
Server Error
Sorry, something went terribly wrong!
0 - syntax error, unexpected ''ละเว้นแฟ้�' (T_ENCAPSED_AND_WHITESPACE)

**Comment by withindale** [comment]
Occurs after submitting logon.

**Comment by withindale** [comment]
PS Stupidly, I did not backup the site before the upgrade so, as I can't login, the easy way would be to replace or modify a file. Thanks.

**Comment by withindale** [comment]
Closed by mistake!

**Comment by rhukster** [comment]
Don’t think it’s the short codes.   The error looks like I escaped utf8 chars in yaml somewhere. What is that Thai?

**Comment by withindale** [comment]
That Thai is on the error page (see first message above).

**Comment by rhukster** [comment]
I would search for that string, and see where it's coming from.  I can't really tell what the error is unless I can see it in context.  Also try setting the `Errors` option to full errors (`1`) in `user/config/system.yaml`. That will provide a full stack trace that will pinpoint where the error is occuring.

**Comment by withindale** [comment]
There is an error at the  end of line 468 in the compiled Thai file, tagged '_ORIG' attached. The 'diamond ?' character translated to a single quote when I tried to pasted it here. As a workround I have replaced the Thai with the Indonesian file. Not sure how the compilation process works or where to look for the origin of the 'diamond ?' character. shown in this message.
0 - syntax error, unexpected ''ละเว้นแฟ้�' (T_ENCAPSED_AND_WHITESPACE)

That character prevents the file from downloading so I removed it from the 'IGNORE_HIDDEN_HELP' string.
